### DISCUSSION TITLE
can we get day of week on clock hover ?

### DISCUSSION THREAD

**Discussion by emoxam** [martinet101/ElevenClock#874] [Q&A]
Hello! Thanks fot the app! 
Can we get day of week on clock hover ?
I feel a lack of it)
Thanks

**Comment by marticliment** [comment]
Hi @emoxam, do you mean like on the clock tooltip?

Can you please open an issue explaining in detail the feature request and how would it behave?

**Comment by emoxam** [comment]
Pardon. Setting up full format of time resolves the issue. Thanks)
